q2_bowtie2/_bowtie.py

```diff
@@ -127,8 +152,20 @@ def align_single(
             str(threads),
         ]
         with tempfile.NamedTemporaryFile() as temp:
-            subprocess.run(cmd, check=True, stdout=temp)
+            result = subprocess.run(cmd, stdout=temp, stderr=subprocess.PIPE)
```

Comment:
Is bowtie really writing results to stderr? That seems so odd

Reply:
Yeah, I was also quite surprised...
They are writing the alignment summary to stderr while they are are writing the alignment file to stdout that you are saving.